index.js

```diff
@@ -145,11 +146,32 @@ function handleError(err) {
   console.error(err);
 }
 
+function getNextTimeout(){
+	var now = new Date();
+	var waitingTime;
+	startTime.setHours(22);
+	startTime.setMinutes(0);
+	waitingTime = startTime.getTime() - now.getTime();
+	console.log('Next search starts in '  + waitingTime/(60*1000) + ' minutes.');
+	return waitingTime;
+}
+
 (function initialize() {
-  var searchInterval = setInterval(search, LIMIT_TIME / LIMIT_GET);
-  var publishInterval = setInterval(publish, LIMIT_TIME / LIMIT_POST);
-  setTimeout(function () {
-    clearInterval(searchInterval);
-    clearInterval(publishInterval);
-  }, LIMIT_TIME);
+	var nextEjecTime = getNextTimeout();
+	if(nextEjecTime < 0){
```

Comment:
Why not have this check inside the "getNextTimeout" method? Inside that method check if waiting time is positive, add a day otherwise.
This way, you can remove this "if" statement altogether and simply set the timeout.